packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -7225,23 +7270,45 @@ export default class Meeting extends StatelessWebexPlugin {
     }
 
     if (tracks.microphone) {
-      this.setLocalAudioTrack(tracks.microphone);
+      const oldTrack = this.mediaProperties.audioTrack;
```

Comment:
would have been nicer if change track happened on the track level we don't have to do on and off

Reply:
maybe, but even when you do that change in webrtc-core and change underlying track when changing microphone, at API level you still need to allow people to just call publish with a completely new LocalMicrophoneTrack, so then you still have to do on and off.